src/Cake.Svn/SvnAliases.Info.cs

```diff
@@ -0,0 +1,255 @@
+        /// <returns>The Subversion information.</returns>
+        /// <example>
+        /// <para>Gets Subversion information about directory at <paramref name="directoryPath"/>.</para>
+        /// <code>
+        /// <![CDATA[
+        ///     var svnInfoSettings = new SvnInfoSettings
+        ///     {
+        ///         Depth = SvnDepth.Unknown;
+        ///     };
+        ///
+        ///     var localDirectoryInfo = GetDirectoryInfo(@"C:\project\src\", svnInfoSettings);
+        ///
+        ///     foreach(var svnInfoResult in localDirectoryInfo)
+        ///     {
+        ///         Verbose("Path: {0}", svnInfoResult.Path);
+        ///     }
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Info")]
+        [CakeNamespaceImport("Cake.Svn.Info")]
+        public static IEnumerable<SvnInfoResult> GetDirectoryInfo(
+            this ICakeContext context, 
+            DirectoryPath directoryPath, 
+            SvnInfoSettings settings)
+        {
+            var svnInformation = new SvnInfo(context.Environment, SvnClientFactoryMethod);
+
+            return svnInformation.GetInfo(directoryPath, settings ?? new SvnInfoSettings());
+        }
+
+        /// <summary>
+        /// Gets Subversion information about the file at <paramref name="filePath"/>.
+        /// </summary>
+        /// <param name="context">The Cake context.</param>
+        /// <param name="filePath">The file.</param>
+        /// <returns>The Subversion information.</returns>
+        /// <example>
+        /// <para>Gets Subversion information about the file at <paramref name="filePath"/>.</para>
+        /// <code>
+        /// <![CDATA[
+        ///     var localFileInfo = GetFileInfo(@"C:\project\src\file.cs");
+        ///
+        ///     Verbose("Path: {0}", localFileInfo.Path);
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Info")]
+        [CakeNamespaceImport("Cake.Svn.Info")]
+        public static SvnInfoResult GetFileInfo(this ICakeContext context, FilePath filePath)
+        {
+            return GetFileInfo(context, filePath, null);
+        }
+
+        /// <summary>
+        /// Gets Subversion information about file at <paramref name="filePath"/>.
+        /// </summary>
+        /// <param name="context">The Cake context.</param>
+        /// <param name="filePath">The file.</param>
+        /// <param name="settings">The settings.</param>
+        /// <returns>The Subversion information.</returns>
+        /// <example>
+        /// <para>Gets Subversion information about file at <paramref name="filePath"/>.</para>
+        /// <code>
+        /// <![CDATA[
+        ///     var svnInfoSettings = new SvnInfoSettings
+        ///     {
+        ///         Depth = SvnDepth.Unknown;
+        ///     };
+        ///
+        ///     var localFileInfo = GetFileInfo(@"C:\project\src\", svnInfoSettings);
```

Comment:
To avoid potential naming conflicts with other addins I suggest to name this `GetSvnFileInfo`